lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -937,6 +937,9 @@ class DefaultAssetPickerBuilderDelegate
   /// Whether the bottom actions bar should display.
   bool get hasBottomActions => isPreviewEnabled || !isSingleAssetMode;
 
+  /// The tap gesture recognizer.
+  TapGestureRecognizer? _tapGestureRecognizer;
```

Comment:
The recognizer is only used to present the limited overlay, and we generally do not write private fields for the delegate. So, consider renaming it to:
```suggestion
  TapGestureRecognizer? presentLimitedTapGestureRecognizer;
```